### ISSUE TITLE
[BITV] 9.1.4.1/7.1 - The selected/active navigation item "Logging" in the left-side navigation has a contrast difference of less than 3:1 with the color of adjacent navigation items. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36947]
Additional emphasis such as bolding or underlining is also missing.

- [ ] Fix

![fe580438a652a6f47479931eb8768873](https://user-images.githubusercontent.com/1315170/222220902-2168e980-5820-49f9-b9b3-0045d6342581.png)

![414c728086658dfdb595a873bd04deeb](https://user-images.githubusercontent.com/1315170/222220917-6d60638b-56c7-4b85-859c-49d8ff6fcd6c.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-45ae33ca9f-v7-n1

</details>

**Comment by JuliaKirschenheuter** [comment]
Dear @michaelnissenbaum,

I've discussed possible implementations with our designers. After several reflections we came to conclusion that the best way would be to add a border with color `--color-primary-element` like:

![Screenshot from 2023-05-15 12-58-06](https://github.com/nextcloud/server/assets/6078378/1b2bd6e5-3a28-4fcd-a7a2-ff4c58119535)
![Screenshot from 2023-05-15 12-57-48](https://github.com/nextcloud/server/assets/6078378/c09e1671-e83d-45d8-a553-50ebd5c4799e)

It that would be be ok, should we take 1px or 2px as border width? 

Another alternative would be to change background color, but designers saying they don't use such a dark color anywhere other than the primary button and on the borders.

Would a solution with border work? Thank you!

**Comment by michaelnissenbaum** [comment]
Hi @JuliaKirschenheuter . Using a border to highlight an active menu element is a good idea. I would lean towards a 2px border. However, it's important to ensure that the border has a minimum contrast ratio of 3:1 with the background. In the screenshot, we have contrasts below 3:1.

**Comment by michaelnissenbaum** [comment]
@JuliaKirschenheuter The solution looks good from my perspective.
